sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -144,131 +128,55 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
         true,
         "Result must be any array for list requests"
       );
-      should.equal(
```

Comment:
There was an odd indentation issue where this test case was nesting instead of being closed. Not sure how it was missed earlier - should be fixed now.